src/constants.ts

```diff
@@ -104,7 +104,7 @@ declare var WALL_HITS: number;
 declare var WALL_HITS_MAX: number;
 
 declare var EXTENSION_HITS: number;
-declare var EXTENSION_ENERGY_CAPACITY: number;
+declare var EXTENSION_ENERGY_CAPACITY: {[level: number]: number};
```

Comment:
Can we provide a more useful name than `level`? `roomControllerLevel` or `rcl` perhaps? I'd prefer the longer one, given that users will never actually have to type the label to the bracketing, and will likely find the descriptive name to be useful.